src/pages/resultsView/styles.scss

```diff
@@ -125,3 +125,13 @@
     width: 100%;
     max-width: 555px;
 }
+
+.menuLine {
+    border-style: solid !important;
+    border-color: #3178ae !important;
+    border-width: 1px !important;
+}
+
+p.styleMyItalicText {
```

Comment:
see comments below.  ithink we can get rid of this css

Reply:
@alisman Gone.
Now this pull request only affects the 2 code files, and removes the logo image.